### ISSUE TITLE
How to add a new unit to security scans

### ISSUE THREAD

**Issue by dekiel** [kyma-project/test-infra#5788]
**Description**

As a Kyma developer I would like to know how I can add a respective unit to security scans to comply with security requirements. I would like to know what I can do myself and for which part I need to ask other team for help. I would like to know which team I need to ask for which part of configuration. I would like to have a step by step like instruction attached.

As a neighbours team member I would like to know what are required steps to add respective unit to security scans. I would like to know which steps are required from my team and how to execute them. I would like to have a step by step like instruction attached.

**Area**

* security scans

**Reasons**

Neighbors team is going to take over responsibility for security scan tools. We need to know exactly how to add end to end a new unit to security scans. We need to have written down instruction which we can reuse for such requests as this will be one of many tools we support. Apart from that we would like to enable other teams to execute some of the steps which do not require neighbours team access levels.

**Comment by dekiel** [comment]
@aberezovski can we have such instructions before you leave or maybe they are already somewhere?

**Comment by Halamix2** [comment]
Manual for Checkmarx is available here: https://wiki.wdf.sap.corp/wiki/display/kyma/Checkmarx+scanning+for+Kyma+source+code

**Comment by aberezovski** [comment]
Thanks @Halamix2 for mentioning the Checkmarx onboarding guide. 
I guess @dekiel was asking about having a centralised place (documentation) that describe the onboarding for all security scans required by SAP. Am I right Przemek?

**Comment by dekiel** [comment]
Yes, my intention was to have easy to access and use documentation for all security tools we use now. If such documentation already exist but it's distributed in multiple places, the solutions might be to migrate them or simply aggregate all needed links in one place with some brief description what they contains. Number of tools, spaces locations etc. makes difficult for users to find documentation they can use.

Apart from that, a user who want to onboard its unit might don't exactly know which tools he must onboard. For Checkmarx probably not all users have rights to add new project and can't use this documentation.

**Comment by aberezovski** [comment]
@dekiel @Halamix2 @Ressetkk
I prepared a [wiki page](https://wiki.one.int.sap/wiki/display/kyma/Kyma+Mandatory+Security+Testing+Tools) that reflects all the mandatory security testing tools which are must be used for Kyma.
Any feedback is welcomed

**Comment by kasiakepka** [comment]
I've asked Alexandru to add information how to add new Whitesource Product and created this simple guide here: https://wiki.one.int.sap/wiki/display/kyma/Add+new+repositories+to+the+mandatory+scanners

**Comment by kasiakepka** [comment]
<details>
  <summary>Please follow ⛔**Release blocker** guidelines</summary>

**Assignment:**
•	 Release blockers can't be left unassigned.
•	 When it is unclear who should be assigned, contact Release Managers or Scrum Masters on #kyma-skr-release.
 
**Investigation:**
•	Split the release blocker into smaller issues if it concerns more teams or more than one bug or error type appears.
•	If the most effective way of working on that issue is to have a group call - use #kyma-team to organize it or ask Scrum Masters to do that.
 
**Time-box:**
•	Update release-blockers every day with valuable information about progress and planned delivery.
•	Contact Release Managers if you are not able to deliver fix before branch cut-off.
•	When issue happens after branch cut-off update it by 1PM every day.

**Escalation:**
•	Inform Release Managers about no progress or existing blockers.
</details>
